frontend/internal-packages/db/supabase/migrations/20250714080634_add_workflow_runs_table.sql

```diff
@@ -0,0 +1,184 @@
+BEGIN;
+
+-- Create workflow_run_status enum type
+CREATE TYPE "public"."workflow_run_status" AS ENUM (
+    'pending',
+    'success',
+    'error'
+);
+
+ALTER TYPE "public"."workflow_run_status" OWNER TO "postgres";
+
+-- Create workflow_runs table to track LangGraph workflow executions
+CREATE TABLE IF NOT EXISTS "public"."workflow_runs" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "design_session_id" "uuid" NOT NULL,
+    "organization_id" "uuid",
+    "run_id" "uuid" NOT NULL,
+    "status" "public"."workflow_run_status" DEFAULT 'pending' NOT NULL,
+    "created_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "updated_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+ALTER TABLE "public"."workflow_runs" OWNER TO "postgres";
+
+-- Add primary key constraint
+ALTER TABLE ONLY "public"."workflow_runs"
+    ADD CONSTRAINT "workflow_runs_pkey" PRIMARY KEY ("id");
+
+-- Add foreign key constraints
+ALTER TABLE ONLY "public"."workflow_runs"
+    ADD CONSTRAINT "workflow_runs_design_session_id_fkey" FOREIGN KEY ("design_session_id") REFERENCES "public"."design_sessions"("id") ON UPDATE CASCADE ON DELETE CASCADE;
+
+ALTER TABLE ONLY "public"."workflow_runs"
+    ADD CONSTRAINT "workflow_runs_organization_id_fkey" FOREIGN KEY ("organization_id") REFERENCES "public"."organizations"("id") ON UPDATE CASCADE ON DELETE RESTRICT;
+
+-- Add unique constraint on run_id
+ALTER TABLE ONLY "public"."workflow_runs"
+    ADD CONSTRAINT "workflow_runs_run_id_key" UNIQUE ("run_id");
+
+-- Add index for better query performance
+CREATE INDEX "workflow_runs_design_session_id_idx" ON "public"."workflow_runs" ("design_session_id");
+CREATE INDEX "workflow_runs_organization_id_idx" ON "public"."workflow_runs" ("organization_id");
+CREATE INDEX "workflow_runs_status_idx" ON "public"."workflow_runs" ("status");
```

Comment:
nits

Since `status` is an enum with only a few possible values(three values: pending, process, error), this index might not offer much performance benefit and could increase write overhead. Consider removing it unless you’ve observed clear query performance gains.